api/models/Events.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  attributes: {
+    operation: {
+      type: 'string',
+      required: true,
+      enum: ['check in']
+    },
+    event_received: {
+      type: 'datetime',
+      required: true
+    },
+    person_id: {
+      type: 'string',
+      required: true,
+      unique: true,
```

Comment:
person id on the event model won't be unique?
don't you want to link that to the detainee where the person_id will be unique?